pkg/oci/oci.go

```diff
@@ -928,7 +928,7 @@ func ensureImage(ctx context.Context, imageStore oras.Target, image string, imgO
 	}
 
 	if !imgOpts.VerifyPublicKey {
-		imgOpts.Logger.Warnf("you set --verify-image=false, image will not be verified")
+		imgOpts.Logger.Warnf("you set option to not verify image, use at your own risk")
```

Comment:
Perhaps something less aggressive towards the users: "image signature verification is disabled"?